packages/flutter/test/widgets/reorderable_list_test.dart

```diff
@@ -717,6 +717,37 @@ void main() {
     expect(tester.getTopLeft(find.text('item 0')), const Offset(0, 500));
   });
 
+  testWidgets('SliverReorderableList - properly animates the drop at starting position', (WidgetTester tester) async {
```

Comment:
Can you add a similar test where `reverse: true`?

Reply:
@Piinks  I have pushed the required test.